subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -134,15 +137,20 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	}
 
 	/* Add ACAD to AUX_SYNC_IND */
-	err = ull_adv_sync_acad_enable(lll_adv_sync,
-				       (sizeof(struct pdu_big_info) + 2),
-				       (void **)&acad);
+	hdr_data.field_data = field_data;
+	field_data[0] = sizeof(struct pdu_big_info) + 2;
```

Comment:
Similar comment ragarding the size of the advertising data header

Reply:
No, there is no define and host too uses magic value 2 (https://github.com/zephyrproject-rtos/zephyr/blob/aec4c4977dbccef2d4486eb37d12ea679cb9d9c8/subsys/bluetooth/host/adv.c#L315)

As a separate task, may be develop some reusable code for AD data population.

Reply:
Since we have no precedence of using a macro value for this, I'll let it pass, but it's nonetheless something we generally should avoid doing in my opinion.